File: kartwheel/app/src/main/java/us/handstand/kartwheel/layout/KartTextView.java
package us.handstand.kartwheel.layout;

import android.content.Context;
import android.support.v7.widget.AppCompatTextView;
import android.util.AttributeSet;

import org.jetbrains.annotations.NotNull;
import org.jetbrains.annotations.Nullable;

public final class KartTextView extends AppCompatTextView {
    public KartTextView(@NotNull Context context) {
        super(context);
    }

    public KartTextView(@NotNull Context context, @Nullable AttributeSet attrs) {
        super(context, attrs);
        KartFontView.INSTANCE.setLineSpacingFromAttrs(this, context, attrs);
        if (!isInEditMode()) {
            KartFontView.INSTANCE.setTypefaceFromAttrs(this, context, attrs);
        }

    }

    public KartTextView(@NotNull Context context, @Nullable AttributeSet attrs, int defStyleAttr) {
        super(context, attrs, defStyleAttr);
        KartFontView.INSTANCE.setLineSpacingFromAttrs(this, context, attrs);
        if (!isInEditMode()) {
            KartFontView.INSTANCE.setTypefaceFromAttrs(this, context, attrs);
        }

    }
}

File: kartwheel/app/src/main/kotlin/us/handstand/kartwheel/layout/KartTextView.kt
package us.handstand.kartwheel.layout

import android.content.Context
import android.support.v7.widget.AppCompatTextView
import android.util.AttributeSet


class KartTextView : AppCompatTextView, KartFontView {
    constructor(context: Context?) : super(context)
    constructor(context: Context?, attrs: AttributeSet?) : super(context, attrs) {
        setLineSpacingFromAttrs(attrs)
        setTypefaceFromAttrs(attrs)
    }

    constructor(context: Context?, attrs: AttributeSet?, defStyleAttr: Int) : super(context, attrs, defStyleAttr) {
        setLineSpacingFromAttrs(attrs)
        setTypefaceFromAttrs(attrs)
    }
}